xla/service/gpu/transforms/windowed_einsum_handler.cc

```diff
@@ -467,6 +467,24 @@ bool ShouldAddToChain(const HloInstruction* inst) {
       return false;
   }
 }
+
+HloComputation* MakeSumComputation(PrimitiveType type, HloModule* module) {
+  HloComputation::Builder sum_b("add");
+  auto x = sum_b.AddInstruction(HloInstruction::CreateParameter(
+      /*parameter_number=*/0, ShapeUtil::MakeShape(type, {}), "x"));
+  auto y = sum_b.AddInstruction(HloInstruction::CreateParameter(
+      /*parameter_number=*/1, ShapeUtil::MakeShape(type, {}), "y"));
+  if (type == PRED) {
+    sum_b.AddInstruction(HloInstruction::CreateBinary(
+        ShapeUtil::MakeShape(type, {}), HloOpcode::kOr, x, y));
```

Comment:
Is summing a group of booleans considered an "Or" operation? In any case, I don't think we'll run this codepath, since we only match Add instructions, not Or instruction, below.

Reply:
Yea since it's only 1 bit, adding bunch of 1s will make it cap at 1 so it's similar to an "or". Anyway i think your point about this being irrelevant is valid. I removed it.